packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import CreateRancher, { CreateRancherProps } from './CreateRancher';
+import { render, waitFor, fireEvent } from '../../../utils/test.provider';
```

Comment:
We should not mix source code and code specific to tests
Let's put this out of src folder and create an alias if you prefer

Reply:
I'm talking about the test.provider.tsx file in the utils folder that is also used to contain utils for source code.